articles/202408041030-git-metrics.md

```diff
@@ -0,0 +1,105 @@
+# Project Metrics
```

Comment:
@jdrouet **suggestion:** « What is the git-metrics project? »

Reply:
I went for "Build metrics and budgets with git-metrics"